script/parse-examples

```diff
@@ -4,6 +4,21 @@ set -e
 
 cd "$(dirname "$0")/.."
 
+function checkout_at() {
+  repo=$1; url=$2; sha=$3
+  if [ ! -d "$repo" ]; then
+    git clone "https://github.com/$url" "$repo"
+    git fetch && git reset --hard "$sha"
```

Comment:
Probably need a `cd` between the `git clone` and the `git fetch`?

Maybe just the `clone` could be inside the conditional, and the `pushd`, `git fetch`, `git reset`, `popd` sequence could happen unconditionally?

Reply:
💥  You implemented this before I suggested it.

Reply:
lol! as you can tell, bash is not my forte